### DISCUSSION TITLE
Microsoft Visual C++ 2015-2022 Redistributable x86 and x64

### DISCUSSION THREAD

**Discussion by dlundahl91** [Romanitho/Winget-Install#105] [General]
Hey team

I don't know how I can do this in a better way, to solve my issues..

In my organisation, I am trying to deploy Microsoft OLE DB Driver for SQL server on to all of our PC's. We have 750 PC's across the organisation.
The way I am doing it is by making an intune package, that is dependent on:
Winget package: Microsoft.VCRedist.2015+.x64
and
Winget package: Microsoft.VCRedist.2015+.x86

If I open a sandbox, and install those two winget packages manually, I get the newest VCRedist versions, and all is OK to install the OLE DB driver.

But when it is done during our autopilot process, it appears that our intune packages (using winget-install) says they are already installed, as sometimes it is already installed with version 14.30.30704.0 for example, and the newest is 14.38.33135.0

The problem here is that the OLE DB driver requires 14.34.*.* or newer, and therefore that package fails.
We have set winget-autoupdate to run 2 days after PC has been autopiloted, and then every 2 days after that, to check if there are updates, but I must find a way to ensure that my VCRedist packages both x64 and x86 install the newest and not approve detection, simply because it is there.
Is it possible somehow to add a specific version as minimum, for the detection script to detect and then --force install the newest version on top?

Thanks so much in advance, I seriously appreciate your help!

Best regards
Dan

**Comment by Romanitho** [comment]
Hi.
We have to work on an option to force install anyway.
Thanks ;)
